pytest_doctestplus/plugin.py

```diff
@@ -119,6 +123,20 @@ def pytest_addoption(parser):
     parser.addoption("--doctest-only", action="store_true",
                      help="Test only doctests. Implies usage of doctest-plus.")
 
+    parser.addoption("--doctest-plus-generate-diff",
+                     help=(
+                         "Generate a diff where expected output and actual "
+                         "output differ.  "
+                         "The diff is printed to stdout if not using "
+                         "`--doctest-plus-generate-diff=overwrite` which "
+                         "causes editing of the original files.\n"
+                         "NOTE: Unless an in-pace build is picked up, python "
+                         "file paths may point to unexpected places. "
+                         "If `"overwrite"` is not used, will create a temporary "
```

Comment:
```suggestion
                         "If 'overwrite' is not used, will create a temporary "
```
Just going to use single ' and no backtick.

Thanks for the other suggestions and fixes, they looked great!